assets/src/scripts/components/hydrograph/scales.js

```diff
@@ -14,9 +15,20 @@ const paddingRatio = 0.2;
  */
 function extendDomain(domainExtent) {
     const padding = paddingRatio * (domainExtent[1] - domainExtent[0]);
+
+    // If all values are above zero, make zero the lower bound
+    if (domainExtent[0] >= 0 && domainExtent[1] >= 0) {
+        return [
+            Math.max(0, domainExtent[0] - padding),
+            domainExtent[1] + padding
+        ];
+    }
+
+    // If we have values less than zero, keep 20% padding around both sides.
```

Comment:
Style thing, but I think having one return statement is cleaner.

Reply:
I like that when there's cleanup to do before returning, but otherwise I tend to prefer the "return early" notion of weeding out the edge cases first.

Reply:
In this case though there really are only have two paths, in which case I think it's clearer to have an else clause.